evap/evaluation/tests/test_misc.py

```diff
@@ -24,13 +24,13 @@ def setUpTestData(cls):
         Degree.objects.filter(name_de="Bachelor").update(import_names=["Bachelor", "B. Sc."])
         Degree.objects.filter(name_de="Master").update(import_names=["Master", "M. Sc."])
 
-    def test_sample_xls(self):
+    def test_sample_xlsx(self):
```

Comment:
```suggestion
    def test_sample_semester_file(self):
```

(or enrollment, whatever floats your boat)